web_ui/packages/config/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@geti/config",
+  "version": "1.0.0",
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "description": "",
+  "exports": {
+    "./test": "./test/jest.config.mjs",
+    "./lint": "./lint/eslint.config.mjs",
+    "./typescript": "./typescript/tsconfig.base.json"
```

Comment:
At first glance it wasn't immediately abvious to me how the other packages were able to import from these files, but I guess that's what the exports file does.
I'm wondering if it might be easier if we name these the same, i..e `import config from '@geti/config/lint/eslint.config.ts` instead? Or if we could add a single `index.ts` file as an entrypoint to the package?

This isn't super important for the config package as that's something we will only look at once in a while. But I think the way we export for say the ui package will be important.